app/models/ability.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :index, :site
+    can [:permalink, :edit, :help, :fixthemap, :offline, :export, :about, :preview, :copyright, :key, :id], :site
```

Comment:
Can these not be merged into one?

Is the ability to use a symbol as the last argument to refer to actions in a controller that does correspond to a model documented anywhere? Everything I've seen just refers to using a class name there to authorize access to objects of that class?

Reply:
I'll merge these, good spot. The symbol usage and `class => false` is from https://github.com/CanCanCommunity/cancancan/wiki/Non-RESTful-Controllers